test/server/cards/02_SHD/units/SupercommandoSquad.spec.js

```diff
@@ -0,0 +1,38 @@
+describe('Supercommando Squad', function() {
+    integration(function() {
+        describe('Supercommando Squad\' ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [{ card: 'supercommando-squad', upgrades: ['shield'] }],
+                    },
+                    player2: {
+                        groundArena: ['wampa', 'jedha-agitator'],
+                    }
+                });
+            });
+            it('should give it sentinel only as long as it is upgraded', function () {
+                this.player1.pass();
+
+                this.player2.clickCard(this.wampa);
+                //Supercommando Squad automatically selected due to sentinel
```

Comment:
nit, code styling: can you add a space after "//", I need to update the linter to catch this